tutorials-dp0-3/portal-dp0-3-3.rst

```diff
@@ -0,0 +1,425 @@
+
+Clicking on the "Apply" button will result in the plot showing the distribution of TNO diameters extracted via your query.  
+
+.. figure:: /_static/portal_tut03_step01l.png
+    :width: 600
+    :name: portal_tut03_step01l
+    :alt: screenshot illustrating the distribution of TNO's diameters
+
+    The screenshot illustrating the distribution of the TNO diameters in your sample, revealing that diameters of TNOs are in the range of a few hundred kilometers.  
+
+
+1.11.  And now for something that I can't explain:  distribution of H magnitudes as a functon of semi-major axis of the orbit.  
+In the previous plot, replace the "plot settings" with "q/(1-e)" for the x-axis, and H magnitude for y axis.  This results in the plot below.    
+
+.. figure:: /_static/portal_tut03_step01m.png
+    :width: 600
+    :name: portal_tut03_step01m
+    :alt: A screenshot of the plot of H magnitude vs. semi-major axis 
+
+Why are there so many objects with H magnitude exactly at 15, over a wide range of heliocentric distances?  
+
+
+1.12. Clear the query and results and return to the RSP TAP Search form.
+
+.. _DP0-3-Portal-3-Step-2:
+
+Step 2. Find and explore a well-observed TNO
+============================================
+
+2.1. Follow steps 1.1 and 1.2 above to navigate to the ADQL query interface, and enter the query below.
+This query has the same basis as the one used above in step 1.2, with three changes.
+One, it joins with the ``DiaSource`` table to retrive the number of ``DiaSources`` (i.e., detections) associated with each object.
+Two, it applies a constraint that the semi-major axis be between 30 and 100 AU.
+Three, it uses a different constraint on ``ssObjectId`` to return a different random subset.
+
+.. code-block:: SQL 
+
+    SELECT mpc.ssObjectId, COUNT(ds.ssObjectId), mpc.e, mpc.q 
+    FROM dp03_catalogs_10yr.MPCORB AS mpc 
+    JOIN dp03_catalogs_10yr.DiaSource AS ds ON mpc.ssObjectId = ds.ssObjectId 
+    WHERE mpc.ssObjectId < -700000000000000000 
+    AND mpc.q > 30 * (1 - mpc.e) 
+    AND mpc.q < 100 * (1 - mpc.e) 
+    GROUP BY mpc.ssObjectId, mpc.e, mpc.q 
+
+This search might take up to a minute.  
+
+
+2.2. The default results view plots the first two columns against each other, ``ssObjectId`` and ``COUNT``,
+which is not particularly useful but it does show the number of detections for the most oft-detected TNOs 
+is in the thousands.
+Click twice on the ``COUNT`` in the table to short descending by count.
+
+.. figure:: /_static/portal_tut03_step02a.png
+    :width: 600
+    :name: portal_tut03_step02a
+    :alt: A screenshot of the default results view with the table sorted by count.
+
+    The default results view from the ADQL query above.
+
+
+2.3. The query returns about 12,600 objects.  
+We will continue with the object with the largest number of observations - 12,103 of them! - with the ``ssObjectId`` = -735085100561880491.  
```

Comment:
Another narrative voice change. Rewrite all instances of "we" and "you", here and below.

Step 2.3 has no action. Try to have one action per step.

Reply:
Done.